packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -632,32 +630,40 @@ export default class Meeting extends StatelessWebexPlugin {
         }
         this.mediaProperties.webrtcMediaConnection.requestMedia(MediaType.VideoMain, mediaRequests);
       }),
-      screenShareAudio: new MediaRequestManager((mediaRequests) => {
-        if (!this.mediaProperties.webrtcMediaConnection) {
-          LoggerProxy.logger.warn(
-            'Meeting:index#mediaRequestManager --> trying to send screenshare audio media request before media connection was created'
-          );
+      screenShareAudio: new MediaRequestManager(
```

Comment:
this block of changes is unrelated, it's just a fix for broken build on beta, it will disappear from the diff once I merge latest beta into this PR (but I can't now, because beta is broken with something else now)